dspace/config/spring/api/bitstore.xml

```diff
@@ -36,6 +38,28 @@
         <property name="subfolder" value="${assetstore.s3.subfolder}"/>
     </bean>
 
+    <!-- Define JCloudStoreService bean -->
+    <!--
+    <util:constant static-field="org.jclouds.filesystem.reference.FilesystemConstants.PROPERTY_BASEDIR"
+                   id="propertyBaseDir"/>
+
+    <bean name="jcloudStore" class="org.dspace.storage.bitstore.JCloudBitStoreService" scope="singleton" lazy-init="true">
+        <property name="enabled" value="${assetstore.s3.generic.enabled:false}"/>
+        <property name="useRelativePath" value="${assetstore.s3.generic.useRelativePath:false}"/>
+        <property name="container" value="${assetstore.s3.generic.container}"/>
+        <property name="endpoint" value="${assetstore.s3.endpoint}"/>
+
+        <property name="providerOrApi" value="${assetstore.s3.generic.provider}"/>
+        <property name="identity" value="${assetstore.s3.generic.awsAccessKey}"/>
+        <property name="credentials" value="${assetstore.s3.generic.awsSecretKey}"/>
+        <property name="maxCounter" value="${assetstore.s3.maxCounter:100}"/>
+
+        Subfolder to organize assets within the bucket, in case this bucket is shared
+        Optional, default is root level of bucket
+        <property name="subFolder" value="${assetstore.s3.generic.subfolder}"/>
```

Comment:
This property name should be `subfolder`, it will cause a spring startup error otherwise

Reply:
I've redone the merge on the latest `main` revisions and made the adjustment to use `subfolder`